web/frontend/README.md

```diff
@@ -0,0 +1,33 @@
+# Automated Tests w/ Playwright
+
+## Setup
+
+Run
+
+```
+npx playwright install-deps --dry-run
+```
+
+to be shown the dependencies that you need to install on your machine (requires `root` access).
+
+You need to have the D-Voting application and DELA network running. You also need to make sure
+that the environment variables from the D-Voting application are set (e.g. `FRONT_END_URL`) in
+the shell you'll be executing the tests in.
+
+## Run tests
+
+Run
+
+```
+npx playwright test
```

Comment:
```suggestion
FRONT_END_URL=http://localhost:3000 npx playwright test
```

Reply:
forgot about these two comments, I'll add it

Reply:
updated the README for both comments